lms/djangoapps/instructor_task/tests/test_api.py

```diff
@@ -140,38 +145,43 @@ def test_submit_reset_all_with_long_url(self):
     def test_submit_delete_all_with_long_url(self):
         self._test_submit_with_long_url(submit_delete_problem_state_for_all_students)
 
-    def _test_submit_task(self, task_function, student=None):
+    @ddt.data(
+        (submit_rescore_problem_for_all_students, 'rescore_problem'),
+        (submit_rescore_problem_for_all_students, 'rescore_problem_if_higher', dict(only_if_higher=True)),
+        (submit_rescore_problem_for_student, 'rescore_problem', dict(student=True)),
+        (submit_rescore_problem_for_student, 'rescore_problem_if_higher', dict(student=True, only_if_higher=True)),
+        (submit_reset_problem_attempts_for_all_students, 'reset_problem_attempts'),
+        (submit_delete_problem_state_for_all_students, 'delete_problem_state'),
+        (submit_rescore_entrance_exam_for_student, 'rescore_problem', dict(student=True)),
```

Comment:
In some cases I prefer this way, especially when the `dict` keys are python variables or field names - rather than string constants.  This way, the code reads better - since the variables are presented in the same form in both places - and especially in cases, such as this, where readability is given precedence over performance.

However, I'm not a stickler on this one, so I'll update.